Specs/Scene/ScreenSpaceCameraControllerSpec.js

```diff
@@ -446,6 +447,63 @@ defineSuite([
         expect(camera.frustum.bottom).toEqual(-camera.frustum.top);
     });
 
+    it('rotate counter-clockwise in 2D', function() {
+        setUp2D();
+        scene.rotatable2D = true;
```

Comment:
I guess this overwrites the property get function with the boolean?  Perhaps it is more obvious to assign to `_ rotatable2D`?